lib/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -120,6 +122,9 @@ public LockActivity() {
         this.options = options;
         this.lockView = lockView;
         this.webProvider = webProvider;
+        this.handler = new Handler();
```

Comment:
These are only used in tests. In absolutely NO scenario Android will instantiate the LockActivity through this constructor, so it's safe to use.